starter-core/src/main/java/io/micronaut/starter/feature/build/gradle/templates/buildGradle.rocker.raw

```diff
@@ -114,35 +133,37 @@ dependencies {
 @if (features.contains("annotation-api")) {
     @dependency.template("javax.annotation", "javax.annotation-api", "implementation", null)
 }
-@if (features.contains("netty-server")) {
-    @dependency.template("io.micronaut", "micronaut-http-server-netty", "implementation", null)
-}
-@if (features.contains("jetty-server")) {
-    @dependency.template("io.micronaut.servlet", "micronaut-http-server-jetty", "implementation", null)
+@if (features.contains("oracle-cloud-sdk")) {
+    @dependency.template("io.micronaut.oraclecloud", "micronaut-oraclecloud-sdk", "implementation", null)
 }
-@if (features.contains("undertow-server")) {
-    @dependency.template("io.micronaut.servlet", "micronaut-http-server-undertow", "implementation", null)
+@if (features.contains("aws-v2-sdk")) {
+    @dependency.template("io.micronaut.aws", "micronaut-aws-sdk-v2", "implementation", null)
 }
-@if (features.contains("tomcat-server")) {
-    @dependency.template("io.micronaut.servlet", "micronaut-http-server-tomcat", "implementation", null)
+@if (features.contains("oracle-function")) {
+    @if (!features.contains("oracle-function-http")) {
+    @dependency.template("io.micronaut.oraclecloud", "micronaut-oraclecloud-function", "implementation", null)
+    @dependency.template("com.fnproject.fn", "api", "implementation", null)
+    @dependency.template("com.fnproject.fn", "runtime", "runtimeOnly", null)
+    @dependency.template("com.fnproject.fn", "testing-junit4", "testImplementation", null)
+    }
 }
 @if (features.contains("google-cloud-function")) {
-    @if (features.contains("google-cloud-function-http")) {
-    @dependency.template("io.micronaut.gcp", "micronaut-gcp-function-http", "implementation", null)
-    } else {
+    @if (!features.contains("google-cloud-function-http")) {
+    @dependency.template("com.google.cloud.functions", "functions-framework-api", "compileOnly", null)
     @dependency.template("io.micronaut.gcp", "micronaut-gcp-function", "implementation", null)
+    @dependency.template("com.google.cloud.functions", "functions-framework-api", "testImplementation", null)
     }
 }
 @if (features.contains("azure-function")) {
+    @if (!features.contains("azure-function-http")) {
     @dependency.template("com.microsoft.azure.functions", "azure-functions-java-library", "implementation", null)
-@if (features.contains("azure-function-http")) {
-    @dependency.template("io.micronaut.azure", "micronaut-azure-function-http", "implementation", null)
-} else {
     @dependency.template("io.micronaut.azure", "micronaut-azure-function", "implementation", null)
-}
+    }
 }
 @if (features.contains("http-client")) {
     @dependency.template("io.micronaut", "micronaut-http-client", "implementation", null)
+} else {
+    @dependency.template("io.micronaut", "micronaut-http-client", "testImplementation", null)
```

Comment:
This should be moved down to the testImplementation section as we currently are ordering deps by scope